admin/src/components/Collections.js

```diff
@@ -0,0 +1,111 @@
+/**
+ *
+ * Block
+ */
+
+import React, { memo, useState, useEffect } from 'react'
+import { request } from 'strapi-helper-plugin'
+import pluginId from '../pluginId'
+import { Table } from '@buffetjs/core'
+import { errorNotifications, successNotification } from '../utils/notifications'
+import { Wrapper } from '../components/Wrapper'
+
+const headers = [
+  {
+    name: 'Name',
+    value: 'name'
+  },
+  {
+    name: 'Status',
+    value: 'status'
+  }
+]
+
+const Collections = () => {
+  const [collectionsList, setCollectionsList] = useState([])
+  const [infoUpdated, setInfoUpdated] = useState(false)
+
+  const updateStatus = async ({ indexUid, updateId }) => {
+    request(`/${pluginId}/indexes/${indexUid}/update/${updateId}`, {
```

Comment:
Pourquoi utiliser ici une promise, alors qu'en dessous tu utilises des async / await ?

Reply:
Because it is a call that could take some time to answer. I call `/update` with an update ID and it will return only when the update has been processed. If the update is waiting on a big document addition it could take a lot of time. Meanwhile I would like my app to continue without being stuck on the await. 

Or should I do it like this : 

```javascript
  const updateStatus = async ({ indexUid, updateId }) => {
     await request()
     ...
     setInfoUpdated(false)
    })
  }

  const addCollectionToMeiliSearch = async ({ name: indexUid }) => {
    const update = await request()
    })
    if (update.error) {
...
    } else {
...
      updateStatus({ indexUid, updateId: update.updateId }) //NO AWAIT
    }
  }
```

Reply:
Ok makes sense. You can keep it like that :)

Reply:
We changed it, thanks!